Cargo.toml

```diff
@@ -28,16 +28,22 @@ anstyle = "1.0.4"
 unicode-width = "0.1.11"
 
 [dev-dependencies]
+annotate-snippets = { path = ".", features = ["testing-colors"] }
```

Comment:
With this you can't test the platform-specific colors in annotate-snippets

Some options
- Offer a "test" constructor used by most tests except for the couple that need platform-specific colors
- Have a test focused solely on the colors (no annotations) that outputs a line per color (with text saying what the semantic meaning is).
  - Something like this is my preferred route as it makes the platform-specific testing (the most annoying to update) be independent of most of the rest of the styling so you can make changes in the output and not have to update platform-specific snapshots

Reply:
I realized this doesn't need to be set here, and should probably be done in a separate PR, when we are actually testing colors